server/base/src/main/java/org/apache/accumulo/server/conf/TableParentConfiguration.java

```diff
@@ -35,6 +36,10 @@ public TableParentConfiguration(String tableId, Instance inst, AccumuloConfigura
 
   @Override
   protected String getNamespaceId() {
-    return Tables.getNamespaceId(inst, tableId);
+    try {
+      return Tables.getNamespaceId(inst, tableId);
+    } catch (TableNotFoundException e) {
+      return null;
```

Comment:
The class should not exist. I'm working on phasing it out as part of the changes for ACCUMULO-4050